scripts/seed.py

```diff
@@ -137,6 +138,9 @@ def create_challenge(title, start_date, end_date, host_team):
     evaluation_script = open(
         os.path.join(settings.BASE_DIR, 'examples', 'example1', 'sample_evaluation_script.zip'), 'rb')
     queue = ''.join(random.choice(string.ascii_letters) for _ in range(75))
+    title = title.lower().replace(" ", "-")
```

Comment:
Can you explain why are you using `.lower().replace(" ", "-")` for `title`? 
It should be used only for `slug` and not `title`.

Reply:
I was thinking it was okay updating title as both title and year make up the slug. But I didn't notice that it will create an issue when Challenge model is being created with the same title. I will resolve the issue soon